p2p/node.go

```diff
@@ -68,9 +68,7 @@ const (
 	chanceToCheckBandwidthUsage = 0.1
 	// defaultMaxBytesPerSecond is the maximum number of bytes per second that a
 	// peer is allowed to send before failing the bandwidth check.
-	// TODO(albrow): Reduce this limit once we have a better picture of what
-	// real world bandwidth should be.
-	defaultMaxBytesPerSecond = 104857600 // 100 MiB.
+	defaultMaxBytesPerSecond = 1048576 // 1 MiB.
```

Comment:
Should we add a new comment back in here?

Reply:
👍Just updated the comments in both places.